pages/learn/develop/runtime.md

```diff
@@ -275,8 +275,22 @@ In the {{ $names.company.lower }} host OS [dnsmasq][dnsmasq-link] is used to man
 
 {{> "general/persistent-storage"}}
 
-### Inconsistency in `/tmp` Directory
-At the time of writing there is an inconsistency in the behavior of `/tmp` directory during reboot and services restart. With the current behavior any thing in `/tmp` will persist over a device reboot, but will **not** persist over services restart.
+### Temporary directories
+
+Note that the `/tmp` and `/var/tmp` directories in a container are not true `tmpfs` volumes by default, and they are treated like any other ephemeral container layers.
```

Comment:
Do we have a standard link for folks to refer to what exactly is a `tmpfs` volume?
I am saying because, I didn't know and since we are mentioning they are `not  true tmpfs volumes`. I think we need them to be certain exactly as to what tmpfs is.

Reply:
This seems nice https://www.kernel.org/doc/html/latest/filesystems/tmpfs.html